tests/test_measurements.py

```diff
@@ -73,6 +76,7 @@ def test_create_invalid_name(self):
         with self.assertRaises(KeyError):
             Measurement(metric, value, metric_set=self.metric_set)
 
+    @unittest.skip('FIXME DM8477 this is failing')
```

Comment:
Woah, this one shouldn't fail. It doesn't fail for me. Ah! Are you getting an error about `UnitTypeError`? That exception was added in a more recent astropy than our current default. It would be good to upgrade to astropy 1.3 anyway, which will bring that in.